tests/testthat/test-dataset_vignette.R

```diff
@@ -1,23 +1,34 @@
 library(admiral.test)
 
-## Test 1: a 'knitr_kable' object is output when run outside pkgdown ----
-test_that("dataset_vignette Test 1: a 'knitr_kable' object is output when run outside pkgdown", {
+# dataset_vignette
+## Test 1: A 'knitr_kable' object is output when run outside pkgdown ----
+test_that("Test 1: A 'knitr_kable' object is output when run outside pkgdown", {
```

Comment:
```suggestion
test_that("dataset_vignette Test 1: A 'knitr_kable' object is output when run outside pkgdown", {
```

We actually want it the other way.  :)  https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#set-up-the-test-script
![image](https://user-images.githubusercontent.com/10111024/195439998-19b745c1-71dc-4b1d-941c-531ce872a72f.png)

Reply:
@zdz2101 , you can easily achieve the correct formatting by calling the "format test_that test file" addin (see https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#addin-admiraldevformat_test_that_file).